packages/codemods/src/codemods-helpers.tsx

```diff
@@ -10,11 +11,39 @@ import {
   Node,
   ImportNamespaceSpecifier,
   Collection,
+  FileInfo,
+  API,
+  Options,
 } from 'jscodeshift';
+
 import { COMPILED_IMPORT_PATH, REACT_IMPORT_PATH, REACT_IMPORT_NAME } from './constants';
+import { CodemodPlugin } from './plugins/contract';
+import DefaultCodemodPlugin from './plugins/default';
 
 type Identifiers = Array<Identifier | JSXIdentifier | TSTypeParameter>;
 
+const getPlugin = async (pluginPath: string | undefined) => {
+  if (pluginPath) {
+    try {
+      const pluginModule = await import(pluginPath);
+      return pluginModule.default;
+    } catch (err) {
+      console.error(
+        chalk.red(`${chalk.bold(`Plugin at path '${pluginPath}' was not loaded`)}\n${err}`)
+      );
+      process.exit(1);
```

Comment:
I would just throw the error here, if you really want the exit code then specify `process.exitCode = 1;`